platform-coop-toolkit/coop-map-directory-index
# feat: add Organization.code_url

## Description

This PR adds a new URLField, `code_url`, to the `Organization` model. This corresponds to a link to the organization's free / libre / open source code, e.g. a GitHub or GitLab organization.

## Steps to test

1. Check out this branch.
2. Run `python manage.py migrate mdi`.
3. Open an Organization for editing in the admin.

**Expected behavior:** A URL field labelled "Code url" follows the "Code availability" field.

## Additional information

I could see that we might be able to infer this from GitHub if present in the `Organization.socialnetworks` data, but I think it makes sense to allow this to be added separately.

## Related issues

Blocks #6.
